packages/backend/src/managers/apiServer.ts

```diff
@@ -26,14 +26,33 @@ import { existsSync } from 'fs';
 import { getFreeRandomPort } from '../utils/ports';
 import * as podmanDesktopApi from '@podman-desktop/api';
 import { readFile } from 'fs/promises';
+import type { ModelsManager } from './modelsManager';
+import type { components } from '../../src-generated/openapi';
```

Comment:
looks like there are failure on the test

Reply:
Fixed